jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -453,8 +453,7 @@ def _extract_from_aperture(self, spectral_cube, uncert_cube, aperture,
             )  # returns an NDDataArray
             # Remove per steradian denominator
             if astropy.units.sr in collapsed_nddata.unit.bases:
-                aperture_area = (self.aperture_area_along_spectral
```

Comment:
is `aperture_area_along_spectral` used anywhere or should it be removed?

Reply:
It's needed for scaling the background flux by the ratio of the background to aperture areas: https://github.com/spacetelescope/jdaviz/blob/c1ae0f2960371ffcc00ae0ba1f454c3592fb5ee8/jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py#L568

The last important place to check is this though: https://github.com/spacetelescope/jdaviz/blob/c1ae0f2960371ffcc00ae0ba1f454c3592fb5ee8/jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py#L516

@gibsongreen – do you think we need that aperture scale factor?

Reply:
Confirmed with @gibsongreen that we should leave that term there.